webrender/src/device.rs

```diff
@@ -927,18 +943,31 @@ impl Device {
         }
     }
 
+    pub fn bind_read_target(&mut self, texture_id: Option<(TextureId, i32)>) {
+        debug_assert!(self.inside_frame);
```

Comment:
I don't think it's necessary, although perhaps we could call it at the start of the frame once to be safe.